rasa/core/featurizers/tracker_featurizers.py

```diff
@@ -177,6 +173,10 @@ def _entity_data(event: UserUttered) -> Dict[Text, Any]:
     def _remove_user_text_if_intent(trackers_as_states: List[List[State]]) -> None:
         """Deletes user text from state dictionaries if intent is present.
 
+        Only featurizing either the intent or user text is currently supported. When
+        both are present in a state, the user text is removed so that only the intent
+        is featurized.
+
```

Comment:
@samsucik @dakshvar22 Does this explanation make sense?

Reply:
I think so.